components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/NotificationConstants.java

```diff
@@ -18,13 +18,15 @@
 
 package org.wso2.carbon.identity.event.handler.notification;
 
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
 public class NotificationConstants {
 
     private NotificationConstants() {
     }
 
     public static final String TEMPLATE_TYPE = "TEMPLATE_TYPE";
-    public static final String CLAIM_URI_LOCALE = "http://wso2.org/claims/locality";
+    public static final String CLAIM_URI_LOCALE = IdentityUtil.getProperty("LocaleClaimUri");
```

Comment:
shall we move the claim fetch part to a private method? 
Assign a default value if config is null.

Reply:
Fixed in commit 571de68b7b4b2b6c45f7a6d35b7b61bd5512c9b4